src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1132,15 +1132,6 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Target Name="ValidateCommandLineProperties"
           AfterTargets="_CheckForInvalidConfigurationAndPlatform" >
-
-    <NETSdkWarning Condition="'$(_SelfContainedWasSpecified)' != 'true' and
-                   '$(_CommandLineDefinedRuntimeIdentifier)' == 'true' and
-                   '$(RuntimeIdentifier)' != '' and
-                   '$(PublishAot)' != 'true' and
-                   '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
-                   $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '6.0'))"
-               ResourceName="SelfContainedOptionShouldBeUsedWithRuntime" />
```

Comment:
I think we probably shouldn't be deleting this yet.  I think we planned to change the behavior so that specifying the RuntimeIdentifier didn't automatically set SelfContained to true.  I think we_ would change this warning when we changed that, but we would probably only make the new logic apply to .NET 8 and higher, so this warning should probably be left in place for .NET 6 and 7.

Reply:
Ah I see, thanks for double checking it. Let me revert that change